lib/cucumber/formatter/html.rb

```diff
@@ -590,7 +590,7 @@ def lines_around(file, line)
           rescue ArgumentError
             return "# Couldn't get snippet for #{file}"
           end
-          min = [0, line-3].max
+            min = [0, line-3].max
```

Comment:
I'm not convinced on the tabbing above for this file. It's hella complicated.

589-592 I think need one tab

Reply:
Yeah, looks like a phantom tab or something. Will fix. Thanks for the lookout!

Reply:
Fixed in 039b5f1 :)